lib/matplotlib/axes/_axes.py

```diff
@@ -475,9 +481,25 @@ def inset_axes(self, bounds, *, transform=None, zorder=5,
 
         return inset_ax
 
-    def indicate_inset(self, bounds, inset_ax=None, *, transform=None,
-            facecolor='none', edgecolor='0.5', alpha=0.5,
-            zorder=4.99, **kwargs):
+    def indicate_inset(
+            self,
+            bounds: Tuple[float, float, float, float],
+            inset_ax: Optional['Axes'] = None,
+            *,
+            transform: Optional[mtransforms.Transform] = None,
+            facecolor='none',
+            edgecolor='0.5',
+            alpha: float = 0.5,
+            zorder: float = 4.99, **kwargs
+        ) -> Tuple[
+            mpatches.Rectangle,
+            Optional[
+                Tuple[
+                    mpatches.ConnectionPatch, mpatches.ConnectionPatch,
+                    mpatches.ConnectionPatch, mpatches.ConnectionPatch
```

Comment:
If you're going to go to the bother of having closing brackets each on their own line (not really sure it's worth it in this case), I would suggest adding trailing commas as well (minimizing diffs, yada yada).